examples/hid-keys-simple.py

```diff
@@ -0,0 +1,74 @@
+# SPDX-FileCopyrightText: 2021 Sandy Macdonald
+#
+# SPDX-License-Identifier: MIT
+
+# Original file was done for Keybow 2040
+# Author: Sandy Macdonald
+
+# Adaptation for RGB Keypad
+# Author: Thierry Chantier
+
+# A simple example of how to set up a keymap and HID keyboard on Keybow 2040.
```

Comment:
Update and simplify comments, to remove confusing Keybow 2040 references

Reply:
I understand but I also wanted to be super clear that I had minimal changes to do on the original work and credit properly Sandy

Reply:
Of course. 
Not speaking of the credits and licence header, but specifically the description of the script
"A simple example of how to set up a keymap and HID keyboard on Keybow 2040."
=> "A simple example of how to set up a keymap and HID keyboard on the Pimoroni RGB Keypad"
and maybe remove the extra directions 
"You'll need to connect Keybow 2040 to a computer, as you would with a regular USB keyboard."
if not, edit the device name as well.